frontend/src/components/Footer.scss

```diff
@@ -0,0 +1,46 @@
+:root {
+  --roseT: rgba(234, 0, 150, 0);
+  --rose: rgba(234, 0, 150, 1);
+  --bleu: rgba(125, 249, 255, 1);
+  --bleuT: rgba(125, 249, 255, 0);
+  font-family: "Roboto black";
+  background-color: #000510;
```

Comment:
One more recommendation: I'd advise to use a consistent method of color-use : either `rgba(r,g,b,a)` *or* hexa code ``#RRGGBBAA